pom.xml

```diff
@@ -126,6 +126,18 @@
 							<outputDirectory>${basedir}/src/main/resources</outputDirectory>
 						</configuration>
 					</execution>
+					<execution>
+						<id>get-matlab-gen-script</id>
+						<phase>validate</phase>
+						<goals>
+							<goal>wget</goal>
+						</goals>
+						<configuration>
+							<url>https://ssd.mathworks.com/supportfiles/ci/matlab-script-generator/v0/matlab-script-generator.zip</url>

```

Comment:
How can we keep the caching issue from happening again?

Reply:
Done.